src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,10 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int factorial = 1;
```

Comment:
remove this variable :)

Reply:
What about this comment? Be carefully :)
"remove this variable :)"

Reply:
Please !!! I'm sorry very much!
I'll be more attentive!